subjects/min_and_max/README.md

```diff
@@ -0,0 +1,41 @@
+## min_and_max
+
+### Instructions
+
+Create a **function** named `min_and_max` that receives three `i32` and returns a `tuple` with the minimum and the maximum number received as input.
+
+```rust
+pub fn min_and_max(nb_1: i32, nb_2: i32, nb_3: i32) -> (i32, i32) {
+}
+```
+
+### Usage
+
+Here is a program to test your function
+
+```rust
+use min_and_max::min_and_max;
+
+fn main() {
+    println!(
+        "Minium and maximum are: ${:?}",
```

Comment:
The `$` is not necessary and should be removed as it can be confused with some mandatory special char

Reply:
True! Plus I found a typo - `Minium` should be `Minimum`